packages/components/src/Heading/Heading.tsx

```diff
@@ -14,7 +14,7 @@ type Props = {
    */
   as?: HeadingElement;
   children: TypographyProps<HeadingElement>["children"];
-  className: TypographyProps<HeadingElement>["className"];
```

Comment:
Was hitting a type error because `className` was a required prop. I don't think we actually want it to be required?